CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+### Changed
```

Comment:
I think this also needs a `## [Unreleased]` above it, similar to https://github.com/puppetlabs/puppetlabs-reboot/pull/215/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL6 - though I don't think we need to do a release for this change.